addons/mod_tool/plugin.gd

```diff
@@ -5,14 +5,14 @@ var dock: Control
 
 
 func _enter_tree() -> void:
+	add_autoload_singleton('ModToolStore', "res://addons/mod_tool/store.gd")
 	dock = preload("res://addons/mod_tool/dock.tscn").instance()
 	dock.editor_interface = get_editor_interface()
 	dock.editor_interface = get_editor_interface()
-	dock.store.load_store()
```

Comment:
Is it not saved and loaded anymore?

Reply:
That should do it: [b47afed](https://github.com/GodotModding/godot-mod-tool/pull/23/commits/b47afedc2bd978bb8eb1a58517f31e6f3f114496)